### PR TITLE
fixing parse OpenAPI document for application error

### PR DESCRIPTION
**Pull request by tt-le**
to address issue: #102 

Error was generated by openapi trying to generate documentation from an empty/commented-out `src/main/webapp/META-INF/openapi.yaml`. The file should be added in a later step.

Removed the file and modified instructions to indicate that this file should be created instead.

### CODE REVIEW

**Review by AustinSeto — CHANGES_REQUESTED** [review]
We should add a `.gitignore` file to the `start/src/main/webapp/META-INF` folder so it is already created for the user.

**Comment by tt-le** [comment]
@AustinSeto Made the change. What do you think?

**Review by AustinSeto — APPROVED** [review]
LGTM

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
@tt-le The paragraph before the command need to be rewritten if change to create instead of update

**Review by griffinhadfield — APPROVED** [review]
Verified in lgdev. LGTM

**Comment by gkwan-ibm** [comment]
hi @Charlotte-Holt would you review the README.adoc?

File: README.adoc [hunk 1]
```diff
@@ -357,26 +357,22 @@ might be fully or partially complete. If the document is fully complete, then yo
...
-[role="edit_command_text"]
-Uncomment all the lines in the [hotspot file=0]`openapi.yaml` file.
 
 openapi.yaml
 [source, text, linenums, role='code_column']
 ----
 include::finish/src/main/webapp/META-INF/openapi.yaml[]
 ----
 
+This document is the same as your current OpenAPI document with additional APIs for the `/inventory/properties` endpoint.
```
Comment (Charlotte-Holt) [inline README.adoc:375]: Suggested update

>This document is the same as your current OpenAPI document with extra APIs for the `/inventory/properties` endpoint.

File: README.adoc [hunk 2]
```diff
@@ -357,26 +357,22 @@ might be fully or partially complete. If the document is fully complete, then yo
 annotation scanning entirely by setting the `mp.openapi.scan.disable` MicroProfile Config property to `true`.
 If the document is partially complete, then you can augment it with code.
 
-You can find a complete OpenAPI document in the [hotspot file=0]`src/main/webapp/META-INF/openapi.yaml` file.
-This document is the same as your current OpenAPI document with additional APIs for the `/inventory/properties`
-endpoint. To make use of this document, open this document in your favorite text editor and uncomment
-all of its lines. 
+To use pregenerated OpenAPI document, create the OpenAPI document yaml file.
```
Comment (Charlotte-Holt) [inline README.adoc:360]: Suggested update

>To use the pre-generated OpenAPI document, create the OpenAPI document YAML file.

**Review by Charlotte-Holt — COMMENTED** [review]
@tt-le Just a few ID review comments to address. Thanks!

**Comment by tt-le** [comment]
@Charlotte-Holt  Thanks for the feedback! Addressed.

File: README.adoc [hunk 3]
```diff
@@ -357,26 +357,22 @@ might be fully or partially complete. If the document is fully complete, then yo
 annotation scanning entirely by setting the `mp.openapi.scan.disable` MicroProfile Config property to `true`.
 If the document is partially complete, then you can augment it with code.
 
-You can find a complete OpenAPI document in the [hotspot file=0]`src/main/webapp/META-INF/openapi.yaml` file.
-This document is the same as your current OpenAPI document with additional APIs for the `/inventory/properties`
-endpoint. To make use of this document, open this document in your favorite text editor and uncomment
-all of its lines. 
+To use the pregenerated OpenAPI document, create the OpenAPI document yaml file.
```
Comment (Charlotte-Holt) [inline README.adoc:360]: @tt-le Thanks for making updates! Could we change "pregenerated" to "pre-generated"? Also "yaml" to "YAML"?

Comment (tt-le) [reply README.adoc]: @Charlotte-Holt Updated!

**Review by Charlotte-Holt — APPROVED** [review]

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**